include/IECoreHoudini/CoreHoudini.h

```diff
@@ -57,7 +59,7 @@ class CoreHoudini
 	public:
 
 		/// This loads hou into the global context
-		static void initPython();
+		static void IECOREHOUDINI_API initPython();
```

Comment:
Any reason not to just mark the whole class with IECOREHOUDINI_API?

Reply:
Just to limit the number of symbols I'm exposing, with a mind of getting rid of this entire thing eventually. I could change it if you prefer though.

Reply:
I think logically the class is part of the API, so it would make more sense to expose the whole thing, even if we want to remove it later.

Reply:
Fixed in 26177e044b44ce9bbcdde02405220876d66db3b4